extensions/expression-language/tests/integration/UpdateAttributeIntegrationTest.cpp

```diff
@@ -38,11 +38,10 @@ class TestHarness : public IntegrationBase {
   }
 
   void runAssertions() override {
-    using org::apache::nifi::minifi::utils::verifyLogLinePresenceInPollTime;
-    assert(verifyLogLinePresenceInPollTime(std::chrono::milliseconds(wait_time_),
+    REQUIRE(minifi::test::utils::verifyLogLinePresenceInPollTime(std::chrono::milliseconds(wait_time_),
         "key:route_check_attr value:good",
         "key:variable_attribute value:replacement_value"));
-    assert(false == verifyLogLinePresenceInPollTime(std::chrono::milliseconds(200), "ProcessSession rollback"));  // No rollback happened
+    REQUIRE(false == minifi::test::utils::verifyLogLinePresenceInPollTime(std::chrono::milliseconds(200), "ProcessSession rollback"));  // No rollback happened
```

Comment:
REQUIRE_FALSE?

Reply:
Updated in 96d3b6a6896c37bb5b493a035a6f258e1bc97ada